frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -13,6 +13,7 @@ local T = require("ffi/util").template
 local ReaderDictionary = InputContainer:new{
     data_dir = nil,
     dict_window_list = {},
+    lookup_msg = _("Dictionary lookup of :\n“%1” ...")
```

Comment:
Yes, but where to put `Dictionary` or `Wikipedia`, and the search term(s) ?
I'd like them to strike in the few ms/seconds the InfoMessage is visible.
`Looking up "19th century in literature" on Wikipedia...` or
`Looking up 'maintainers' in Dictionary...`
with a possible wrap in the middle, make it hard to catch the selected text.